Makefile

```diff
@@ -14,7 +14,7 @@ RESET := $(shell tput -T linux sgr0)
 TITLE := $(BOLD)$(PURPLE)
 SUCCESS := $(BOLD)$(GREEN)
 # the quality gate lower threshold for unit test total % coverage (by function statements)
-COVERAGE_THRESHOLD := 50
+COVERAGE_THRESHOLD := 47
```

Comment:
this is surprising, that even after adding tests it is dropping

Reply:
the problem is that the go tooling does not count packages that have 0 tests. I added tests to a package that previously had no tests, so now the rest of the package is being counted. So though coverage is technically going up, it's being counted down :( (this is odd, but has happened a couple times before as well).